### ISSUE TITLE
Release a precompiled version

### ISSUE THREAD

**Issue by ghost** [Ovyl/tabouli#9]
I think that it would be great to have a precompiled version of this app, either available via the release page of github or in some main distro repositories (but that has some work to it).
In my use case I don't intend to install the go toolchain on my system and it would be great to have a precompiled package that I can just move in my bin folder.

**Comment by nicksinas** [comment]
@sriveco Definitely. We are going to start with a providing compiled versions via a GitHub Release. We are also looking into Homebrew for macOS. What OS are you running?

**Comment by ghost** [comment]
Either ArchLinux or WSL on windows. But x86 on both so should work without problems.
